src/ol/source/Source.js

```diff
@@ -66,6 +67,12 @@ class Source extends BaseObject {
      */
     this.attributions_ = this.adaptAttributions_(options.attributions);
 
+    /**
+     * @private
+     * @type {boolean}
+     */
+    this.attributionsCollapsible_ = options.attributionsCollapsible;
```

Comment:
You could default this to `true` here, then other subclasses do not need to be changed.